### ISSUE TITLE
Bug: blank view

### ISSUE THREAD

**Issue by kawamikasa** [Esri/feedback-js-api-next#65]
Hi,

Can someone point me to this type of error please: `_can not read property "data" of null SpriteMosaic.js_`

![Capture](https://user-images.githubusercontent.com/70646069/91990308-dd49c080-ed31-11ea-8516-b90ae91c071b.PNG)

Randomly this happens during graphic update in a graphicslayer and I can not reproduce it with a simple test. View becomes blank and white (background I supposed).

Api version : 4.15

Thanks

**Comment by bsvensson** [comment]
@kawamikasa - please share the simple test in a codepen or similar.  Also, please try with `next` version. Thank you.

**Comment by kawamikasa** [comment]
@bsvensson  
I'm sorry, I misspoke, I can't reproduce it in a simple test.
Impossible to upgrade to 4.16 at this moment because of this : https://community.esri.com/thread/257490-unable-to-extend-featurelayer-at-416 - Hug refactoring work

**Comment by bsvensson** [comment]
Is the problem with a vanilla GraphicsLayer or related to your extended featurelayer mentioned in that geonet post?

Can you share your code for the "graphic update" that causes this problem?

**Comment by kawamikasa** [comment]
> Is the problem with a vanilla GraphicsLayer or related to your extended featurelayer mentioned in that geonet post?

class CustomGraphicsLayer extends GraphicsLayer { // props - methods - ... }
I'dont think it's related to that
Just, it was to tell that at the moment we cannot migrate to 4.16

> Can you share your code for the "graphic update" that causes this problem?

It's just a graphicsLayer.graphics.remove(graphic); graphicsLayer.graphics.add(newGraphic); 

If I refresh application it's work fine, I have all graphics

PS : 
- Graphics symbol is a CIMSymbol
- if I remove CIMTextSymbol part, It works

```
const symbol = {
  type: "cim",
  data: {
    type: "CIMSymbolReference",
    symbol: {
      type: "CIMPointSymbol",
      symbolLayers: [
        {
          type: 'CIMVectorMarker',
          enable: true,
          size : 32,
          frame, **// calculate frame based on text size**
          markerGraphics: [
            {
              type: 'CIMMarkerGraphic',
              geometry: { x: 0, y: 0 },
              symbol: {
                type: 'CIMTextSymbol',
                fontFamilyName: 'Arial Unicode MS',
                fontStyleName: 'Normal',
                height: 16,
                kerning: true,
                horizontalAlignment: 'Center',
                offsetX: 0,
                offsetY: 0,
                blockProgression: 'TTB',
                symbol: {
                  type: 'CIMPolygonSymbol',
                  symbolLayers: [
                    {
                      type: 'CIMSolidFill',
                      enable: true,
                      color: [0, 125, 26, 255],
                    },
                  ],
                },
                verticalAlignment: 'Center',
              },
              textString: `${text}`,  // text variable from graphic attributes
            },
            {
              type: 'CIMMarkerGraphic',
              geometry: {
                rings: [
                  [
                    [-5, 5],
                    [5, 5],
                    [5, -5],
                    [-5, -5],
                    [-5, 5],
                  ],
                ],
              },
              symbol: {
                type: 'CIMPolygonSymbol',
                symbolLayers: [
                  {
                    type: 'CIMSolidFill',
                    enable: true,
                    color: [178, 178, 178, 255],
                  },
                ],
              },
            },
          ],
          scaleSymbolsProportionally: true,
          respectFrame: true,
        },
      ],
    },
  },
};
```

**Comment by annelfitz** [comment]
Hi @kawamikasa - thanks for reporting this! This error should be fixed in the next build of `/next`, which will be released at version 4.18
